src/actions/tokens.example.js

```diff
@@ -1,11 +0,0 @@
-/**
```

Comment:
Have you looked into Travis's [Encrypting environment variables](https://docs.travis-ci.com/user/environment-variables/#Encrypting-environment-variables)

I think it is the easiest way to handle APIs with travis.